tests/unittests/map/test_factory.py

```
@@ -98,12 +110,15 @@ def test_invalid(self):
                 )
                 #####
 
-        @pytest.mark.parametrize("batch_method", ("id", "slice", None))
+        @pytest.mark.parametrize(
+            "batch_method", ("id", "id_range", "slice", None)
```

Ruff (0.15.9)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)